devops/resources/check-lifecycle-hooks.sh

```diff
@@ -12,7 +12,7 @@ cd $WORKSPACE/configuration
 . util/jenkins/assume-role.sh
 
 cd $WORKSPACE/sysadmin
-pip install -r requirements.txt
+pip install -r requirements/base.txt
```

Comment:
We may have also wanted to update the comment on https://github.com/edx-ops/sysadmin/blob/master/requirements/constraints.txt#L11